### DISCUSSION TITLE
margin-bottom to place legends

### DISCUSSION THREAD

**Discussion by ljegou** [neocarto/bertin#97] [Ideas]
Hello, I'm using a fairly square geolayer and I'd like to specify a bottom margin of the plot to place the legend, instead of the 4 sides "margin" parameter. Thanks !
![Capture d’écran 2022-12-06 à 08 19 20](https://user-images.githubusercontent.com/5158753/205846477-b91421dc-fc6f-4cd0-a914-7a47e37ec593.png)

**Comment by neocarto** [comment]
Hello Laurent. It is not possible for the moment but it's a good idea. Please add an issue.
